core/src/main/java/org/frankframework/util/flow/MermaidFlowGenerator.java

```diff
@@ -84,6 +84,7 @@ public MermaidFlowGenerator() {
 	@Override
 	public void afterPropertiesSet() throws Exception {
 		String frankElementsList = compileFrankElementList();
+		log.debug("Initialized frankElementList to be [{}]", frankElementsList);
```

Comment:
Misschien is generated beter? Ik zie dat het nuttig is om dit te loggen maar gezien de grote lap tekst die in de log verschijnt is trace net iets handiger.
```suggestion
		log.trace("generated frankElementList [{}]", frankElementsList);
```